src/eko/matching_conditions/operator_matrix_element.py

```diff
@@ -170,65 +224,58 @@ def quad_ker(
             basis function configuration
         a_s : float
             strong coupling, needed only for the exact inverse
+        nf: int
+            number of active flavor below threshold
         L : float
             :math:`log(q^2/m_h^2)`
         backward_method : ["exact", "expanded" or ""]
-            empty or method for inverting the matching contidtion (exact or expanded)
+            empty or method for inverting the matching condition (exact or expanded)
         is_msbar: bool
             add the |MSbar| contribution
     Returns
     -------
         ker : float
             evaluated integration kernel
     """
-    is_singlet = mode0 in [100, 21, 90]
-    # get transformation to N integral
-    r = 0.4 * 16.0 / (1.0 - logx)
-    if is_singlet:
-        o = 1.0
-        indices = {21: 0, 100: 1, 90: 2}
-    else:
-        o = 0.0
-        indices = {200: 0, 91: 1}
-    n = mellin.Talbot_path(u, r, o)
-    jac = mellin.Talbot_jac(u, r, o)
+    ker_base = QuadKerBase(u, is_log, logx, mode0)
+    integrand = ker_base.integrand(areas)
+    if integrand == 0.0:
+        return 0.0
 
     # compute the harmonics
     sx = np.zeros(3, np.complex_)
```

Comment:
the original idea here was to have `sx` with a *fixed* size and to fill the elements when needed - now, already in the old code this was broken as PTO >= 1 already used a reassignment (`sx = np.array(...)`); however, I'd say let's make it consistent now (since with N3LO you're adding more and more elements)

Reply:
Will be fixed in #110